static/bidder-params/iqzone.json

```diff
@@ -0,0 +1,15 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "title": "IQZone Adapter Params",
+    "description": "A schema which validates params accepted by the IQZone adapter",
+    "type": "object",
+  
+    "properties": {
+      "placementId": {
+        "type": "string",
+        "description": "Placement ID"
+      }
+    },
+  
+    "required": ["placementId"]
```

Comment:
Just to double-check: Are we sure every request will come with a `"placementId"` parameter? If it's set as `"required"`, an incoming request with a missing `"placementId"` will get rejected and Prebid Server won't send a bid request to any bidder. So adding a required field can break the entire setup of a publisher.